### ISSUE TITLE
Tux Doesn't Have Reaction Time for Jumps Anymore

### ISSUE THREAD

**Issue by weluvgoatz** [SuperTux/supertux#1151]
Windows 10, SuperTux ~0.6.0.  

In previous updates, Tux could still jump a little bit after leaving a block, leaving space for human reaction time.  This has been removed, which makes some jumps not possible at all anymore.  Please re-add this or at least consider re-adding it.  I remember this being first reported by AllytheOcelot in the SuperTux discord but he didn't report it, also the issue exists for me too.

**Comment by HybridDog** [comment]
I've also noticed that Tux doesn't jump off the ledge, even with #1158 applied.
Now I've recorded a video with visible collision boxes and keyboard input. When I looked at it frame by frame, the keyboard input was shown too late. Maybe there is latency in the input recognition or maybe I've just reacted too slowly. It happens with and without vsync. I hope that I haven't misunderstood this issue
[video-2019-09-06_17.19.12.mp4.zip](https://github.com/SuperTux/supertux/files/3584603/video-2019-09-06_17.19.12.mp4.zip)
.
